### ISSUE TITLE
Problem using includes with dom elements on some devices

### ISSUE THREAD

**Issue by Lambo232** [qodesmith/datepicker#34]
It appears that some devices do not support the use of includes for looking for a dom element.

In the debug version on line 644     
```javascript
const onCal = path.includes(calendar);  
```
will not work on some devices.

A simple fix is     
```javascript
var onCal = calendar.contains(target);  
```
which seems to work on all devices I have tested.

**Comment by qodesmith** [comment]
@Lambo232 I'll have to dig in a big. Very interesting.

**Comment by qodesmith** [comment]
Finally got around to looking at this. You're right. I didn't realize that DOM nodes had the `contains` method on them. That's a much better solution and I was able to get rid of more code too! This is fixed in the latest version, 3.2.1.
